picoquictest/tls_api_test.c

```diff
@@ -7155,6 +7155,9 @@ int bad_cnxid_test()
             ret = -1;
         }
         else {
+            for (size_t i = 0; i < test_ctx->nb_test_streams; i++) {
+                test_api_delete_test_stream(&test_ctx->test_stream[i]);
+            }
```

Comment:
These frees are needed, but they seems out of place here...

Reply:
Yes. They belong in the deletion of the test context.